lib/tasks/fincap.rake

```diff
@@ -208,38 +267,49 @@ namespace :fincap do
         {{ cms:page:teaser3_image }}
         {{ cms:page:teaser3_text }}
         {{ cms:page:teaser3_link }}
-        {{ cms:page:regional_strategy_title }}
-        {{ cms:page:regional_strategy_text }}
-        {{ cms:page:regional_strategy_link }}
+        {{ cms:page:strategy_title }}
+        {{ cms:page:strategy_overview }}
+        {{ cms:page:strategy_link }}
+        {{ cms:page:steering_group_title }}
+        {{ cms:page:steering_group_links }}
         {{ cms:page:download:simple_component/[Text Link](https://moneyadviceservice.org.uk/link) }}
       CONTENT
     )
 
-    english_site.layouts.find_or_create_by(
-      identifier: 'homepage',
-      label: 'Homepage',
+    puts 'Resetting regional_strategy Layout...'
+    layout = Comfy::Cms::Layout.find_by(identifier: 'regional_strategy')
+    layout.update(
       content: <<-CONTENT
-    {{ cms:page:content:rich_text }}
-    {{ cms:page:hero_image:simple_component/https://moneyadviceservice.org.uk/image.jpg }}
-    {{ cms:page:hero_description:simple_component/Description }}
-    {{ cms:page:teaser_section_title }}
-    {{ cms:page:teaser1_title }}
-    {{ cms:page:teaser1_image }}
-    {{ cms:page:teaser1_text }}
-    {{ cms:page:teaser1_link }}
-    {{ cms:page:teaser2_title }}
-    {{ cms:page:teaser2_image }}
-    {{ cms:page:teaser2_text }}
-    {{ cms:page:teaser2_link }}
-    {{ cms:page:teaser3_title }}
-    {{ cms:page:teaser3_image }}
-    {{ cms:page:teaser3_text }}
-    {{ cms:page:teaser3_link }}
-    {{ cms:page:horizontal_teaser_title }}
-    {{ cms:page:horizontal_teaser_image }}
-    {{ cms:page:horizontal_teaser_text }}
-    {{ cms:page:horizontal_teaser_link }}
-  CONTENT
+        {{ cms:page:content:rich_text }}
+        {{ cms:page:hero_image:simple_component/https://moneyadviceservice.org.uk/image.jpg }}
+        {{ cms:page:hero_description:simple_component/Description }}
+        {{ cms:page:teaser_section_title }}
+        {{ cms:page:teaser1_title }}
+        {{ cms:page:teaser1_image }}
+        {{ cms:page:teaser1_text }}
+        {{ cms:page:teaser1_link }}
+        {{ cms:page:teaser2_title }}
+        {{ cms:page:teaser2_image }}
+        {{ cms:page:teaser2_text }}
+        {{ cms:page:teaser2_link }}
+        {{ cms:page:teaser3_title }}
+        {{ cms:page:teaser3_image }}
+        {{ cms:page:teaser3_text }}
+        {{ cms:page:teaser3_link }}
+        {{ cms:page:strategy_title }}
+        {{ cms:page:strategy_text }}
+        {{ cms:page:strategy_link }}
+        {{ cms:page:forum_title}}
+        {{ cms:page:forum_links}}
+        {{ cms:page:download:simple_component/[Text Link](https://moneyadviceservice.org.uk/link) }}
+      CONTENT
     )
+    puts 'Resetting blocks identifiers...'
```

Comment:
Are this and the following statements going to be executed every time the `fincap:setup` task is invoked? Shouldn't it be just a one time task to run and fix the identifier name?